{"name": "test_30_status_code_404", "status": "passed", "steps": [{"name": "Открытие главной страницы сайта", "status": "passed", "start": 1664801158783, "stop": 1664801161079}, {"name": "Открытие примера Status code 404", "status": "passed", "start": 1664801161079, "stop": 1664801161652}], "attachments": [{"name": "stdout", "source": "ab9a7807-3841-4aac-93c5-2ff87040e541-attachment.txt", "type": "text/plain"}], "start": 1664801158783, "stop": 1664801162342, "uuid": "331938f5-cb3e-4b6e-ab1d-756679906137", "historyId": "89e00f5b2f4b5cac6f36082340ee7eaf", "testCaseId": "2d8cb8f8f6e524d6c1a8c24a8a8eaba1", "fullName": "pom.tests.test_diplom#test_30_status_code_404", "labels": [{"name": "feature", "value": "Проверка что страница возвращает status code 404 "}, {"name": "suite", "value": "Status code 404"}, {"name": "tag", "value": "@pytest.mark.usefixtures('text_inside_test')"}, {"name": "parentSuite", "value": "pom.tests"}, {"name": "host", "value": "DESKTOP-I5N6LU0"}, {"name": "thread", "value": "53864-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pom.tests.test_diplom"}]}